presto-product-tests/conf/presto/etc/singlenode-ldap.properties

```diff
@@ -23,7 +23,9 @@ http-server.https.enabled=true
 http-server.https.keystore.path=/etc/ldap/coordinator.jks
 http-server.https.keystore.key=testldap
 authentication.ldap.enabled=true
-authentication.ldap.url=ldaps://ldap-server:636
-authentication.ldap.ad-domain=presto.testldap.com
-authentication.ldap.base-dn=OU=Asia,DC=presto,DC=testldap,DC=com
-authentication.ldap.group-dn=CN=FinanceSubGroup,OU=America,DC=presto,DC=testldap,DC=com
+
+authentication.ldap.server-type=OPENLDAP
+authentication.ldap.user-object-class=inetOrgPerson
+authentication.ldap.url=ldap://10.25.150.158:389
+authentication.ldap.base-dn=ou=Users,dc=10.25.150.158
+authentication.ldap.group-dn=cn=ldapgroup,ou=Users,dc=10.25.150.158
```

Comment:
Is this IP address of OpenLDAP server temporary until we have OpenLDAP on docker?